person.rb

```diff
@@ -15,7 +16,11 @@ def correct_name
 
   attr_reader :id
 
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
+
+  def add_rentals(date, book)
+    @Rental.new(date, self, book)
```

Comment:
- [ ] Please kindly delete the `@` before the `Rental`, `Rental` is a class and not a variable 😁 😉